english/hammertime/hammertime.md

```diff
@@ -0,0 +1,2 @@
+You break into the hammertime-dance, but quickly realize that this
+doesn't really get to [anywhere](https://github.com/udacity/create-your-own-adventure/blob/master/english/eating-walls/eating-marshmallows.md) 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.